Add more links, some language cleanup (#227)
vespa-engine/blog: _posts/2021-06-18-pretrained-transformer-language-models-for-search-part-4.md

- In the last phase, the top ranking 24 passages from the previous phase are evaluated by the cross attention model. 
- The final ranking score is a linear combination of all three ranking scores. The *rerank-count* can also be adjusted by a query parameter 

> Observe that reusing scores from the previous ranking phases does not impact serving performance a
> s they are only evaluated once (per hit) and cached. 

Reviewer: ```suggestion
Observe that reusing scores from the previous ranking phases does not impact serving performance,
as they are only evaluated once (per hit) and cached. 
```
